fuzz/db_fuzzer.cc

```diff
@@ -0,0 +1,150 @@
+#include <fuzzer/FuzzedDataProvider.h>
+
+#include "rocksdb/db.h"
+
+enum OperationType {
+  kPut,
+  kGet,
+  kDelete,
+  kGetProperty,
+  kIterator,
+  kSnapshot,
+  kOpenClose,
+  kColumn,
+  kCompactRange,
+  kSeekForPrev,
+  OP_COUNT
+};
+
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t* data, size_t size) {
+  rocksdb::DB* db;
+  rocksdb::Options options;
+  options.create_if_missing = true;
+  rocksdb::Status status = rocksdb::DB::Open(options, "/tmp/testdb", &db);
+  if (!status.ok()) {
+    return 0;
+  }
+  FuzzedDataProvider fuzzed_data(data, size);
+
+  // for random string generation
+  const uint8_t* curr_offset = data;
+  size_t curr_size = size;
+
+  std::string value;
```

Comment:
this is only used in https://github.com/facebook/rocksdb/pull/7674/files#diff-e122f366795372b5894f731340ea5650ebc89180a69b86330bb963981cba8adbR49, can we define it just before that line?